src/main/java/tech/gruppone/stalker/server/model/db/OrganizationRoleDao.java

```diff
@@ -0,0 +1,33 @@
+package tech.gruppone.stalker.server.model.db;
+
+import java.time.LocalDateTime;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Builder
+@Data
+@Table("OrganizationRole")
+public class OrganizationRoleDao {
+
+  @NonNull
+  @Column("organizationId")
+  Long organizationId;
+
+  @NonNull
+  @Column("userId")
+  Long userId;
+
+  @Id
+  @Column("administratorType")
+  Long administratorType;
+
+  String name;
```

Comment:
Nota mia: non è un campo della tabella OrganizationRole, però grazie a questa aggiunta l'endpoint risponde correttamente come specificato nel contratto

Reply:
è una soluzione validissima ma io nella branch `implement-organization-users-roles` l'ho fatto in un altro modo (con un enum). proviamo a ragionare su quale dei due modi è meglio usare

questa è la mia implementazione:

https://github.com/GruppOne/stalker-server/blob/8628175169d38b00ca8edd9f25a3122a3671c697/src/main/java/tech/gruppone/stalker/server/model/db/OrganizationRole.java

e questo è l'enum AdministratorType:

https://github.com/GruppOne/stalker-server/blob/8628175169d38b00ca8edd9f25a3122a3671c697/src/main/java/tech/gruppone/stalker/server/model/AdministratorType.java